src/backend.rs

```diff
@@ -399,9 +399,9 @@ impl State {
                 // Ensure that we release the borrow before calling `cb`.
                 let borrow = &*service.borrow();
                 let view = ServiceView::new(borrow);
-                matches = selectors.iter().find(|selector| {
+                matches = selectors.iter().any(|selector| {
```

Comment:
That's a new rule coming from the linter. https://github.com/Manishearth/rust-clippy/wiki#search_is_some